usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -9126,7 +9245,14 @@ static CK_RV ep11_ende_crypt_init(STDLL_TokData_t * tokdata, SESSION * session,
         free(ep11_state);
         goto done;
     case CKR_FUNCTION_NOT_SUPPORTED:
-        /* fallback into ep11 path */
+        /* if mechanism is AES XTS, return error
+         * else fallback to ep11 path */
```

Comment:
Style: can you put this comment into one line? 

Multi line comments should be like this:
```
/*
 * comment ...
 */
```